### PR TITLE
Thomas chen submission

### PR DESCRIPTION
**Pull request by thomasthechen**
Final Validation loss: 4.8935 

https://api.wandb.ai/links/aqlab/8gx0m7in

Adjusted Learning rate and warmup/cooldown timesteps in the cosine learning rate.

### CODE REVIEW

**Comment by nelson-liu** [comment]
thanks!
